en/docs/install-and-setup/setup/setting-up-databases/data-dictionary/api-management-core-related-tables.md

```
@@ -0,0 +1,514 @@
+# API Management Core Related Tables
+
+This section lists out all the API management core related tables and their attributes in the WSO2 API Manager database.
+
+---
+
+### AM_API
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)